tests/docker-compose.yml

```diff
@@ -2,7 +2,7 @@
 # Development configuration of dependent services for Docker Compose.
 #
 postgresql:
-    image: postgres:9.4
+    image: postgres:9.6
```

Comment:
Will also require a bump on Jenkins and in production. This is a breaking change, add it to changelog as such.